streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentInfoTest.java

```diff
@@ -36,80 +36,110 @@
 public class AssignmentInfoTest {
     private final List<TaskId> activeTasks = Arrays.asList(
         new TaskId(0, 0),
-        new TaskId(0, 0),
-        new TaskId(0, 1), new TaskId(1, 0));
+        new TaskId(0, 1),
+        new TaskId(1, 0),
+        new TaskId(1, 1));
     private final Map<TaskId, Set<TopicPartition>> standbyTasks = new HashMap<TaskId, Set<TopicPartition>>() {
         {
+            put(new TaskId(1, 0),
+                Utils.mkSet(new TopicPartition("t1", 0), new TopicPartition("t2", 0)));
             put(new TaskId(1, 1),
                 Utils.mkSet(new TopicPartition("t1", 1), new TopicPartition("t2", 1)));
-            put(new TaskId(2, 0),
-                Utils.mkSet(new TopicPartition("t3", 0), new TopicPartition("t3", 0)));
         }
     };
-    private final Map<HostInfo, Set<TopicPartition>> globalAssignment = new HashMap<HostInfo, Set<TopicPartition>>() {
+    private final Map<HostInfo, Set<TopicPartition>> activeAssignment = new HashMap<HostInfo, Set<TopicPartition>>() {
+        {
```

Comment:
We shouldn't use double-brace initialization. There's a `mkMap` utility function to use instead.

Reply:
I tend to play along with how files are.  esp when I am new to this code base. This is how master is.. Will change these methods. I am using mkMap/mkEntry at other places.. its convenient.